analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/CollectorTestCheck.cs

```diff
@@ -78,5 +78,22 @@ public void ValidateTag(string tag, Action<SymbolicValue> action)
 
         public void ValidateExitReachCount(int expected) =>
             exitReachedCount.Should().Be(expected);
+
+        public void ValidateExecutionCompleted() =>
+            executionCompletedCount.Should().Be(1);
+
+        public void ValidatePostProcess(int expected) =>
+            postProcessed.Should().HaveCount(expected);
+
+        public void ValidateOperationsAreNotNull()
+        {
+            foreach (var context in postProcessed)
+            {
+                context.State[context.Operation].Should().NotBeNull();
+            }
+        }
```

Comment:
Can be simplified to arrow method like this:
```suggestion
        postProcessed.Should().OnlyContain(x => x.State[x.Operation] != null);
```